llvm/lib/IR/Verifier.cpp

```diff
@@ -3632,8 +3632,11 @@ void Verifier::visitCallBase(CallBase &Call) {
     Check(Callee->getValueType() == FTy,
           "Intrinsic called with incompatible signature", Call);
 
+  // Find the actual CC of the callee from the Module.
+  CallingConv::ID CalleeCC = Call.getParent()->getParent()->getParent()
+      ->getFunction(Call.getCalledFunction()->getName())->getCallingConv();
```

Comment:
This is a very complicated way to spell `Callee->getCallingConv()` -- but note that Callee may be null here for indirect calls -- which is the cause of your four thousand or so test failures.